### ISSUE TITLE
missing latex packages: siunitx, minted

### ISSUE THREAD

**Issue by grollinger** [dwrensha/sharelatex#3]
[EDIT: Original Issue Title: Sandstorm package does not support lualatex]
Could you please add that ?

**Comment by dwrensha** [comment]
Thanks for reporting this!

I just published a new version in which I believe this is fixed. https://apps.sandstorm.io/app/5vuv7v0w7gu20z72m78n83rx9qqtqpmtk32f39823wh967z226qh

Could you re-install (or wait for an auto-update), retry, and let me know whether it works?

**Comment by grollinger** [comment]
Yes, lualatex now works.

However... it doesn't seem to be able to find some latex packages.

For example, i am using siunitx which does not work.

Should it be able to get new packages?

**Comment by dwrensha** [comment]
No, there's currently no way for you to add latex packages as a user; I need to include them when I package the app. It looks like it'll be pretty easy for me to include texlive-science, which includes siunitx. Are there any other packages that you'll need?

**Comment by grollinger** [comment]
Below are the packages i currently use in my document.
Most of them are probably already supported.

I also use the Consolas font, which might be hard to add, since it's a windows font and i don't know about the licencing... but doesn't hurt to ask, does it?

In any case, thanks for the immediate assistance :)

% Packages
\usepackage{polyglossia}
\usepackage{siunitx}
\usepackage[bookmarks]{hyperref}

% is this even compatible with ShareLaTeX? ( also needs pygments 
\usepackage[newfloat]{minted} 

\usepackage{caption}
\usepackage{subcaption}
\usepackage[skins, minted, breakable]{tcolorbox}
\usepackage[xindy]{glossaries}
\usepackage{cleveref}
\usepackage{nameref}
\usepackage{csquotes}
\usepackage{tikz}

**Comment by dwrensha** [comment]
I'm not sure about the Consolas font. I think it should be possible to include those packages, including minted, as they do seem to work at sharelatex.com. It looks like it's going to take a bit of configuration, though, and I don't have time to work out the details right at this moment. I'll keep this issue updated.

**Comment by Finkregh** [comment]
this could be renamed to "allow adding latex-packages", so we dont have to open a new issue for each latex-package :-)

**Comment by theNinetyninth** [comment]
How bad would it be to add every package from CTAN?

I have a permanently growing list of packages I use... 
Could it be possible to add packages just like images per Project?

I think it should be possible to convert .sty files to .tex files and \input{} them in the main.tex file. There are few differences to normal LaTeX-code, (I'm only aware of the different use of the '@'-char but maybe I'm wrong.) Could this be a easy solution?

**Comment by xet7** [comment]
This issue was moved to sandstormports/sharelatex#1
